src/package.json

```diff
@@ -32,6 +32,7 @@
 		"crypto": "^1.0.1",
 		"deep-object-diff": "^1.1.9",
 		"dotenv": "^10.0.0",
+		"elevate-cloud-storage": "^2.6.3",
```

Comment:
This was removed right?

Reply:
@nevil-mathew I am not sure , do we have to remove it , i merged the 3.0 release and in this file ,there was a conflict here , i accepted both changes .